azure-pipelines.yml

```diff
@@ -59,11 +59,19 @@ stages:
       product: $(product)
       chartEnv: $(environment)
       preIntegrationTest: true
-      functionalTest: false
+      functionalTest: true
       azureVault: $(vault)
-      azureSecreKeys: 'mi-staging-storage-connection-string'
+      azureSecreKeys: 'mi-staging-storage-connection-string,mi-export-storage-connection-string,mail-username,mail-password,archive-password'
```

Comment:
do you need the connection strings?

Reply:
Was testing with it. Will remove them once I verify the Managed Identity works now with the latest merge.